exercises/concept/port-palermo/.docs/instructions.md

```diff
@@ -12,10 +12,10 @@ The system has to handle identifiers for ships, but also for destinations.
 The first thing you need to do is to create the identifier for the port of Palermo.
 The identifier are the first four letters of the name of the port, in uppercase.
 
-Define the `Port.Identifier` constant to be a symbol with the value `:PALE`.
+Define the `Port::IDENTIFIER` constant to be a symbol with the value `:PALE`.
     
 ```ruby
-Port::Identifier 
+Port::IDENTIFIER 
```

Comment:
@meatball133 thoughts?  Should we not have it as a comment, but have it as explicit code instead?

Reply:
Do you mean like in the stub file, that we give them the constant but let them define it by themself?

Reply:
Yeah.  Because what is presented here is different than how they would likely define it, though by adding an equals sign and a value to this exact syntax, it could be assigned.  I am not sure how important it is to tell them to define the constant if we give them that line in the code... seems like "check the box" rather than "see how a constant is defined", compared to this instruction set and having them find out how to define a constant in that name space.

It might be more straight forward just to do it for them.  It is not a complex idea.

Reply:
Yeah, I am fine either, the goal is to teach how to define a symbol. So in a way, there is no need to have the constant part, but that is some repetition on how to do it.